waku/v2/node/peer_manager/peer_manager.nim

```diff
@@ -284,3 +287,38 @@ proc connectToNodes*(pm: PeerManager,
   # This issue was known to Dmitiry on nim-libp2p and may be resolvable
   # later.
   await sleepAsync(chronos.seconds(5))
+
+# Ensures a healthy amount of connected relay peers
+proc relayConnectivityLoop*(pm: PeerManager) {.async.} =
+  let defaultInterval = chronos.seconds(30)
```

Comment:
Is this a parametrizing constant? Can you move it outside of the proc? To the top of the module with the rest of the constants.

Reply:
yup thanks, done.